packages/flutter_tools/lib/src/compile.dart

```diff
@@ -193,6 +200,8 @@ class KernelCompiler {
           'entryPoint': mainPath,
           'trackWidgetCreation': trackWidgetCreation.toString(),
           'linkPlatformKernelIn': linkPlatformKernelIn.toString(),
+          'engineHash': engine.version,
+          'buildersUsed': '${flutterProject != null ? await flutterProject.hasBuilders : false}',
```

Comment:
https://github.com/flutter/flutter/pull/29399 makes `hasBuilders` synchronous, so you don't need to `await` if you submit that PR first.

Reply:
That one needs to wait on another FE server update, so I'll handle it before submitting there